bitbots_vision/src/bitbots_vision/vision_modules/fcnn_handler.py

```diff
@@ -12,11 +12,12 @@
 class FcnnHandler(CandidateFinder):
     """
     This handles FCNNs, meaning it finds and rates candidates in their output.
+    The FCNN handler runs the FCNN neural network and manages its predictions.
```

Comment:
```suggestion
    The FCNN handler runs the FCNN and manages its predictions.
```
The NN in FCNN means neural network